src/map/modifier.h

```diff
@@ -800,7 +801,6 @@ enum class Mod
 
     // The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
     // 570 through 825 used by WS DMG mods these are not spares.
-    // SPARE = 964, // stuff
```

Comment:
status.lua's mods and spares should 100% match modifier.h's mods and spares. I should not see any different numbers for mods or for spares in either file. when a mod gets refactored out its ID should go back into spares. Topaz contributors what have'yall been doing? These are out of sync again. If there's IDs used by another branch that we're working around there needs to be notes on the PR letting reviewers know this. @zircon-tpl this is a recurring problem. if the numbers do not match it leads people to accidentally use the same one for 2 different things. see previous discord conversation with ibm on the subject.


@tankfest you used 936 but removed spare 964? if we're up to 964 936 should already be in use by something, but I don't see anything using either number so I am confused. And see above general note.

Reply:
Well, what we need to do is stop loading static data from the database, and then these could be generated by the code, ie loading item stats from lua. That's outside the scope of this PR for sure but just a general comment to maybe get some creative juices flowing.

Reply:
@m241dan ibm had a plan and I don't think any contributor currently has both the time and the technical knowhow to make it happen now (by "it" I mean something better than copying what currencies do, which is do get/set with a string value..) but as a future thing I agree that should happen.

Reply:
Used the previously annotated SPARE = 964, updated the next 3 spare mods.